storage/blob_test.go

```diff
@@ -269,6 +269,30 @@ func (s *StorageBlobSuite) TestBlobCopy(c *chk.C) {
 	c.Assert(b, chk.DeepEquals, body)
 }
 
+func (s *StorageBlobSuite) TestStartBlobCopy(c *chk.C) {
+	if testing.Short() {
+		c.Skip("skipping blob copy in short mode, no SLA on async operation")
+	}
+
+	cli := getBlobClient(c)
+	cnt := randContainer()
+	src := randName(5)
+	dst := randName(5)
+	body := []byte(randString(1024))
+
+	c.Assert(cli.CreateContainer(cnt, ContainerAccessTypePrivate), chk.IsNil)
+	defer cli.deleteContainer(cnt)
+
+	c.Assert(cli.putSingleBlockBlob(cnt, src, body), chk.IsNil)
+	defer cli.DeleteBlob(cnt, src, nil)
+
+	// given we dont know when it will start, can we even test destination creation?
+	// will just test that an error wasn't thrown for now.
```

Comment:
Ok, after reading some tests from [other SDKs](https://github.com/Azure/azure-storage-net/blob/68c3ee55a3a6f62a0159cea58005d3fe027312a8/Test/ClassLibraryCommon/Blob/CopyBlobTest.cs#L177), what you can do is try to abort the copy operation when you are sure the operation has ended... It should throw status code [409](https://msdn.microsoft.com/en-us/library/azure/jj159098.aspx#Anchor_3). Sorry for taking so long to answer.